### PR TITLE
Add commands for creating branches and switch branches

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,8 @@
 - `git push -u origin main` / `git push`
 - `git remote -v`
 - `git remote add origin <ssh link>`
+
+## Git Branches & PRs
+
+- `git switch -c <branch-name>`
+- `git switch <branch-name`
```
Comment (ahohnsen) [inline README.md]: Nice, good to know these commands!

**Review by ahohnsen — COMMENTED** [review]
Well done!

File: README.md [hunk 2]
```diff
@@ -9,3 +9,8 @@
 - `git push -u origin main` / `git push`
 - `git remote -v`
 - `git remote add origin <ssh link>`
+
+## Git Branches & PRs
```
Comment (SylvieDoumet) [inline README.md]: We could choose a title without &

File: README.md (hunk 1 above)
Comment (AbdulrahmanTabbakh) [reply README.md]: i find also it s well. Thank u

File: README.md [hunk 3]
```diff
@@ -9,3 +9,8 @@
 - `git push -u origin main` / `git push`
 - `git remote -v`
 - `git remote add origin <ssh link>`
```
Comment (SylvieDoumet) [inline README.md:11]: We could use a html link

File: README.md (hunk 1 above)
Comment (IsabellUrban) [inline README.md]: Am Ende fehlt >

**Review by rvoll — CHANGES_REQUESTED** [review]
You could add the missing bracket. :)

**Review by AbdulrahmanTabbakh — COMMENTED** [review]
Thank u.

**Review by IsabellUrban — COMMENTED** [review]
Looks great!

File: README.md (hunk 2 above)
Comment (Sannsfilm) [inline README.md]: **well done!!** @ahohnsen

File: README.md (hunk 1 above)
Comment (eneben) [inline README.md]: Well done, thank you. You could also add `git switch -` to switch to the branch where we were before. :-)

File: README.md (hunk 3 above)
Comment (ghost) [inline README.md:11]: Great work!

File: README.md [hunk 4]
```diff
@@ -9,3 +9,8 @@
 - `git push -u origin main` / `git push`
```
Comment (laurasliwa) [inline README.md:9]: Important command, especially when you push for the first time!

**Review by eneben — APPROVED** [review]

**Review by laurasliwa — APPROVED** [review]
Important command, especially when you push for the first time!

File: README.md (hunk 1 above)
Comment (P-Oliomonte) [inline README.md]: Es könnte auch noch der Befehl `git switch -` hinzufügt werden = zum letzten Branch switchen

File: README.md (hunk 3 above)
Comment (SylvieDoumet) [inline README.md:11]:
```suggestion
- `git remote add origin <html link>`
```

**Review by SylvieDoumet — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
Great work!

**Review by P-Oliomonte — CHANGES_REQUESTED** [review]
Nur ein kleiner Änderungsvorschlag :-)

File: README.md (hunk 3 above)
Comment (ahohnsen) [reply README.md:11]: True but I leave it like this cause we always want to use the ssh link.

File: README.md (hunk 2 above)
Comment (ahohnsen) [reply README.md]: Thank you!

**Pull request merged**